### DISCUSSION TITLE
自动后退

### DISCUSSION THREAD

**Discussion by Tpfdd** [ywmoyue/biliuwp-lite#755] [bug]
### Bug 描述

多按几下后退键就会开始疯狂自己后退。
还有一个，输入弹幕时后退光标，视频也会跟着后退。
### 操作步骤

```plain text
没有按后退自己一直后退，很容易触发。
```

### 期望结果

修复bug

### 应用来源

Github/其他Github镜像站

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
感谢反馈
